docs/languages/csharp/delegates-strongly-typed.md

```diff
@@ -1,11 +1,96 @@
 # Strongly Typed Delegates
 
+By [Bill Wagner](https://github.com/BillWagner)
+
 [Previous](delegate-class.md)
 
-This article will cover the classes in the .NET framework
-that support delegates, and how those map to the `delegate`
-keyword.
+In the previous article, you saw that you create specific delegate
+types using the `delegate` keyword. 
+
+Delegates are much more useful by embracing and enforcing type
+safety for the methods that are added to the invocation list for
+a delegate object.
+
+In practice, this would lead to creating new delegate types
+whenever you need a different method signature. This work could get tedious
+after a time. Every new feature requires new delegate types.
+
+Thankfully, this isn't necessary. The .NET Core framework contains several
+types that you can reuse whenever you need delegate types. These are
+[generic](generics.md) definitions so you can declare customizations
+when you need new method declarations. 
+
+The first of these types is the [`Action`](http://dotnet.github.io/api/System.Action.html) type, and several variations:
+
+```cs
+public delegate void Action();
+public delegate void Action<in T>(T arg);
+public delegate void Action<in T1, in T2>(T1 arg1, T2 arg2);
+// Other variations removed for brevity.
+```
+
+The `in` modifier on the generic type argument is covered in the article
+on covariance.
```

Comment:
Let's merge this now and we'll add a note to issue #72 to remind us about adding this link.